src/main/java/com/aws/greengrass/clientdevices/auth/PermissionEvaluationUtils.java

```diff
@@ -33,6 +36,11 @@ public final class PermissionEvaluationUtils {
             String.format(SERVICE_RESOURCE_FORMAT, SERVICE_PATTERN_STRING, SERVICE_RESOURCE_TYPE_PATTERN_STRING,
                     SERVICE_RESOURCE_NAME_PATTERN_STRING), Pattern.UNICODE_CHARACTER_CLASS);
 
+    private static final String POLICY_VARIABLE_FORMAT = "\\$\\{iot:(Connection.Thing.ThingName)}";
```

Comment:
shouldn't this "format" be more generic?

Reply:
I originally had it match all variables in the xx.Namespace.Attribute format (with `"\\$\\{iot:(\\w*+.\\w*+.\\w*+)}"`). Joe suggested I only match on Connection.Thing.ThingName for now to keep it simple. Should I change it back?